app/views/patients/_abortion_information.html.erb

```diff
@@ -43,8 +43,15 @@
           <%= f.check_box :solidarity,
                           label: t('patient.abortion_information.clinic_section.solidarity'),
                           label_class: 'tooltip-header-checkbox',
-                          data: { 'tooltip-text': solidarity_help_text } %>
+                          data: { 'tooltip-text': solidarity_help_text } %>            
         <% end %>
+        <%= f.select :solidarity_lead,
+            solidarity_leads,
+            {
+              label: t('patient.abortion_information.clinic_section.solidarity_lead'),
+              label_class: 'tooltip-header-input',
+            },
+            data: {'tooltip-text': 'dogs' } %>
```

Comment:
Slightly cursed because bootstrap select has slightly different behavior. ([github issue](https://github.com/bootstrap-ruby/bootstrap_form/issues/103), [code](https://github.com/bootstrap-ruby/bootstrap_form/blob/13af7bdf07bddd95685293a5e42d006ec0e0bad7/lib/bootstrap_form/inputs/select.rb)). rather than taking a single `options` hash, `select` takes an `options` hash _and_ an `html_options` hash. don't fully understand why, but the data attributes need to go in the latter.

Reply:
<img width="336" alt="Screen Shot 2022-08-16 at 18 57 21" src="https://user-images.githubusercontent.com/7085805/184992412-694bf4f5-3368-4434-b2d1-bdc003279b8c.png">